lib/benchee/system.ex

```diff
@@ -20,6 +20,14 @@ defmodule Benchee.System do
       cpu_speed: cpu_speed()
     }
 
+    unless all_protocols_consolidated?() do
+      IO.puts("""
+      Not all of your protocols have been consolidated. In order to achieve the
+      best possible accuracy for benchmarks, please ensure protocol
+      consolidation is enabled in your benchmarking environment.
+      """)
```

Comment:
I think I'd extract this into its own method, maybe just the `puts` maybe the whole thing. Takes up so much space that makes it seem like a more important piece then it actually is right now.

Maybe a method like `warn_about_performance_degrading_settings` or something? not sure

Reply:
works for me!